lib/lz4hc.h

```diff
@@ -357,6 +375,8 @@ int LZ4_compress_HC_extStateHC_fastReset (void* state, const char* src, char* ds
  */
 LZ4LIB_API void LZ4_attach_HC_dictionary(LZ4_streamHC_t *working_stream, const LZ4_streamHC_t *dictionary_stream);
 
+LZ4LIB_API int LZ4HC_isDirtyContext(const LZ4_streamHC_t *stream);
```

Comment:
What's the value in exposing this to users? As I see it, a primary goal of this project is to hide complexity from the user, so they don't need to care what state the context is in.

Reply:
Agree, it was used in unit tests only. Now it should be removed.